lib/pos_blockchain.go

```diff
@@ -841,7 +828,7 @@ func (bc *Blockchain) commitBlock(blockHash *BlockHash) error {
 	// Connect a view up to the parent of the block we are committing.
 	utxoView, err := bc.getUtxoViewAtBlockHash(*block.Header.PrevBlockHash)
 	if err != nil {
-		return errors.Wrapf(err, "runCommitRuleOnBestChain: Problem initializing UtxoView: ")
+		return errors.Wrapf(err, "commitBlock: Problem initializing UtxoView: ")
```

Comment:
Should this function be called `commitBlockPoS` to match the naming pattern of other PoS blockchain functions?

Reply:
Let's address this rename later in the stack

Reply:
addressed in #790